llm/generate/gen_linux.sh

```diff
@@ -60,12 +60,24 @@ install
 # Placeholder to keep go embed happy until we start building dynamic CPU lib variants
 touch ${BUILD_DIR}/lib/dummy.so
 
-if [ -d /usr/local/cuda/lib64/ ]; then
+if [ -d /usr/local/cuda/lib64/ ] || [ -n ${NIX_CUDA_OUT_DIR} ] || [ -n ${NIX_CUDA_LIB_DIR} ]; then
```

Comment:
Could you follow a pattern similar to ROCM [here](https://github.com/jmorganca/ollama/blob/main/llm/generate/gen_linux.sh#L85-L88)

Check for a $CUDA_PATH, and if not set, use `/usr/local/cuda` as a default.  Then on nix, set CUDA_PATH.

Reply:
Sure! Sounds reasonable to me.